src/Sylius/Bundle/ThemeBundle/Controller/ThemeScreenshotController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ThemeBundle\Controller;
+
+use Sylius\Bundle\ThemeBundle\Model\ThemeInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\HttpFoundation\BinaryFileResponse;
+use Symfony\Component\HttpFoundation\File\Exception\FileNotFoundException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+final class ThemeScreenshotController
+{
+    /**
+     * @var RepositoryInterface
+     */
+    private $themeRepository;
+
+    /**
+     * @param RepositoryInterface $themeRepository
+     */
+    public function __construct(RepositoryInterface $themeRepository)
+    {
+        $this->themeRepository = $themeRepository;
+    }
+
+    /**
+     * @param int $themeId
+     * @param int $screenshotNumber
+     *
+     * @return BinaryFileResponse
+     */
+    public function streamScreenshotAction($themeId, $screenshotNumber)
```

Comment:
What about restricting the convention like:
```
/screenshots/main.jpg
/screenshots/backend-product.jpg
/screenshots/frontend-product.jpg
```
 * Only one-level directory
 * Code is the same as filename (+extension?)